packages/ozone/src/tag-service/embed-content-type-tagger.ts

```diff
@@ -0,0 +1,45 @@
+import { AppBskyEmbedImages } from '@atproto/api'
+import { langLogger as log } from '../logger'
+import { ContentTagger } from './content-tagger'
+
+export class EmbedContentTypeTagger extends ContentTagger {
+  tagPrefix = 'embed-content-type:'
+  isApplicable(): boolean {
+    return Boolean(
+      this.subjectStatus &&
+        this.subject.isRecord() &&
+        this.subject.uri.includes('app.bsky.feed.post') &&
+        !this.subjectStatus.tags?.find((tag) => tag.includes(this.tagPrefix)),
```

Comment:
Just a little more specific!
```suggestion
        !this.subjectStatus.tags?.some((tag) => tag.startsWith(this.tagPrefix)),
```